settings/index.js

```diff
@@ -1,28 +1,56 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import Settings from '@folio/stripes-components/lib/Settings';
-import { stripesShape } from '@folio/stripes-core/src/Stripes'; // eslint-disable-line import/no-unresolved
 
 import ChangePassword from './ChangePassword';
 
 class MyProfile extends Component {
   static propTypes = {
-    stripes: stripesShape.isRequired,
+    stripes: PropTypes.shape({
```

Comment:
I'm interested in why you declared the Stripes-object property by hand in this version, instead of using `stripesShape`?

Reply:
This is because here in that component I have more references on stripes properties than defined in `stripesShape`. There two possible choices here - extend stripes shape or write my proptypes.

Reply:
Thought it might be that. I've been meaning for a while to fix `stripesShape` to include all relevant properties. If I do that, would you go back to using it?

Reply:
Sure.

Reply:
OK, I'll do that. See https://issues.folio.org/browse/STCOR-236

Reply:
This is ready to go, as of stripes-core v2.10.5.

Reply:
Will do.